### PR TITLE
fix #3

### LINKED ISSUE #3: Dev func2
Development

### PR DESCRIPTION
**Pull request by liu770807152**
This is branch file2
how to run
how to test

Signed-off-by: YI <liu770807152@hotmail.com>

### CODE REVIEW

File: file3 [hunk 1]
```diff
@@ -0,0 +1 @@
+file3
```
Comment (126ium) [inline file3:1]: It should be file3.3

**Review by 126ium — CHANGES_REQUESTED** [review]
Fix it plz

File: file3 (hunk 1 above)
Comment (liu770807152) [reply file3:1]: Will do

**Comment by 126ium** [comment]
LGTM

**Review by 126ium — APPROVED** [review]
LGTM

**Pull request merged**